### ISSUE TITLE
Different finetune loss given policy.type=pi0 / policy.path=lerobot/pi0_base. What is the difference?

### ISSUE THREAD

**Issue by BBBBBBob** [huggingface/lerobot#2543]
Hi, I have two different configurations:
1. `	--dataset.repo_id=BBBBBBob/libero_goal_lerobot \
	--dataset.root=/home/j84403411/data/libero/libero_goal_lerobot \
	--policy.path=lerobot/pi0_base \
	--policy.push_to_hub=false \
	--policy.use_proprio=true \
	--output_dir=/home/j84403411/checkpoint/libero/pi0/libero_goal_proprio \
       --policy.dtype=bfloat16 \
       --steps=40_000 \
       --batch_size=16 \
       --rename_map='{"observation.images.image":"observation.images.base_0_rgb",  "observation.images.wrist_image":"observation.images.left_wrist_0_rgb"}' \ `
and
2.
 `	--dataset.repo_id=BBBBBBob/libero_goal_lerobot \
	--dataset.root=/home/j84403411/data/libero/libero_goal_lerobot \
	--policy.type=pi0 \
        --policy.pretrained_path=lerobot/pi0_base \
	--policy.push_to_hub=false \
	--policy.use_proprio=true \
	--output_dir=/home/j84403411/checkpoint/libero/pi0/libero_goal_proprio \
       --policy.dtype=bfloat16 \
       --steps=40_000 \
       --batch_size=16 \
       --policy.input_features='{"observation.state": {"type": "STATE", "shape": [8]},
		"observation.images.wrist_image": {"type": "VISUAL", "shape": [3, 256, 256]},
	 	"observation.images.image": {"type": "VISUAL", "shape": [3, 256, 256]},
		}' \
       --policy.output_features='{"action": {"type": "ACTION", "shape": [7]}}' \ `

The loss trained from the second configuration is 10 times higher than the first one. What caused the difference? Do you know if different checkpoints are loaded in this case? I appreciate your help!
